src/data/favourites/favourites.js

```diff
@@ -8,6 +8,7 @@ const favArray = [
     address: "32 Camden Hill, Eeling, London",
     price: "£££££",
     image: " INSERT IMAGE HERE ",
+    category: "Food & Drink",
   },
   {
     id: "1",
```

Comment:
should these be `_id` ?

Reply:
was a mock array from before we had the id issue, updated to _id